project-apps-dev/ProgrammersOnly/env_files/monog.env

```diff
@@ -0,0 +1,2 @@
+MONGO_INITDB_ROOT_USERNAME=root
```

Comment:
Misspelling in the file name `monog.env` => `mongo.env`

Reply:
Thanks.